docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_9.md

```diff
@@ -0,0 +1,126 @@
+# ExperimentalDetectronGenerateProposalsSingleImage {#openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_9}
+
+**Versioned name**: *ExperimentalDetectronGenerateProposalsSingleImage-9*
+
+**Category**: *Object detection*
+
+**Short description**: The *ExperimentalDetectronGenerateProposalsSingleImage* operation computes ROIs and their scores
+based on input data.
+
+**Detailed description**: The operation performs the following steps:
+
+1.  Transposes and reshapes predicted bounding boxes deltas, variances and scores to get them into the same order as the
+anchors.
+2.  Transforms anchors into proposals using deltas, variances and clips proposals to an image.
+3.  Sorts all `(proposal, score)` pairs by score from highest to lowest; order of pairs with equal scores is undefined.
```

Comment:
We need to clarify this undefined behavior on different frameworks (TF, Caffe2, PDPD) and make clarity for this case for sure because in some corner cases ROIs of the same score can be skipped with `pre_nms` or `post_nms` attributes and we get misalignment with FWs